lib/orderbook/OrderBook.ts

```diff
@@ -816,6 +816,23 @@ class OrderBook extends EventEmitter {
     return order;
   }
 
+  public removeOwnOrders = () => {
+    const successfullyRemovedOrderIds = [];
+    const failedToRemovedOrderIds = [];
+
+    for (const localId of Array.from(this.localIdMap.keys())) {
+      try {
+        this.logger.debug(`Trying to remove order with id: ${localId}`);
+        this.removeOwnOrderByLocalId(localId, true);
+        successfullyRemovedOrderIds.push(localId);
+      } catch (err) {
```

Comment:
I don't think try/catch really makes sense here, since we should expect to be able to remove our own orders. The errors that could arise here would apply only if we try to remove an order that doesn't exist, but if we're removing the set of existing orders that shouldn't be an issue. We shouldn't be hitting exceptions here.